whatshap/cli/polyphase.py

```diff
@@ -981,6 +1002,12 @@ def add_arguments(parser):
         help="Ignore read groups in BAM/CRAM header and assume all reads come "
         "from the same sample.",
     )
+    arg(
+        "--include-haploid-sets",
+        default=False,
+        action="store_true",
+        help="Includes the phase set information for every single haplotype in a custom VCF format field 'HS'.",
```

Comment:
```suggestion
        help="Include the phase set information for every single haplotype in a custom VCF format field 'HS'.",
```

Reply:
I’d also like to see some documentation for new features, but we haven’t really enforced this in the past, so it would be ok for me to omit it now. But generally, I think we should avoid that the only documentation for a feature is the help text on the command line (except in obvious cases).